tests/acceptance/expected-failures-localAPI-on-OCIS-storage.md

```diff
@@ -62,7 +62,7 @@ The expected failures in this file are from features in the owncloud/ocis repo.
 #### [Share lists deleted user as 'user'](https://github.com/owncloud/ocis/issues/903)
 - [apiGraph/deleteGroup.feature:68](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiGraph/deleteGroup.feature#L68)
 
-#### [Renaming the group with = character in it doesn't work](https://github.com/owncloud/ocis/issues/5969)
+#### [Updating group displayName request seems OK but group is not being renamed](https://github.com/owncloud/ocis/issues/5977)
```

Comment:
Should be https://github.com/owncloud/ocis/issues/5969? was recently changed in PR https://github.com/owncloud/ocis/pull/5971
```suggestion
#### [Renaming the group with = character in it doesn't work](https://github.com/owncloud/ocis/issues/5969)
```

Reply:
5969 is closed issue. I created new and link tests